cmd/snap-exec/main_test.go

```diff
@@ -98,15 +103,31 @@ func (s *snapExecSuite) TestFindCommand(c *C) {
 	info, err := snap.InfoFromSnapYaml(mockYaml)
 	c.Assert(err, IsNil)
 
+	snapPath := fmt.Sprintf("%s/snapname/unset", dirs.SnapMountDir)
+
 	for _, t := range []struct {
-		cmd      string
-		expected string
+		app          string
+		cmd          string
+		includeChain bool
+		expected     string
 	}{
-		{cmd: "", expected: `run-app cmd-arg1 $SNAP_DATA`},
-		{cmd: "stop", expected: "stop-app"},
-		{cmd: "post-stop", expected: "post-stop-app"},
+		// With command chain
+		{app: "app", cmd: "", includeChain: false, expected: fmt.Sprintf("%s/run-app cmd-arg1 $SNAP_DATA", snapPath)},
```

Comment:
You can skip `includeChain: false` as `false` is a default for bool fields. Same with `cmd`.

Reply:
Done.